File: (8 kyu) Basic Mathematical Operations/(8 kyu) Basic Mathematical Operations.cpp
int basicOp(char op, int val1, int val2) {
  // Your code here
  if (op == '+'){
    return val1 + val2;
  } else if (op == '-'){
    return val1 - val2;
  } else if (op == '*'){
    return val1 * val2;
  } else if (op == '/'){
    return val1 / val2;
  }
}

/*
int basicOp(char op, int val1, int val2) {
  switch (op) {
    case '+': return val1 + val2;
              break;
    case '-': return val1 - val2;
              break;
    case '*': return val1 * val2;
              break;
    case '/': return val1 / val2;
              break;
    
  }
  return 0;
}
*/

File: (8 kyu) Basic Mathematical Operations/(8 kyu) Basic Mathematical Operations.py
def basic_op(operator, value1, value2):
    #your code here
    try:
        if operator == "+":
            return value1 + value2
        elif operator == "-":
            return value1 - value2
        elif operator == "*":
            return value1 * value2
        elif operator == "/":
            return value1 / value2
    except:
        return None
     
